tests/run-make/branch-protection-check-IBT/rmake.rs

```diff
@@ -4,17 +4,16 @@
 // python3 x.py test --target x86_64-unknown-linux-gnu  tests/run-make/branch-protection-check-IBT/
 
 //@ only-x86_64
+//@ only-stable
 
-//@ ignore-test
 // FIXME(jieyouxu): see the FIXME in the Makefile
```

Comment:
```suggestion
```

Reply:
I wrote `If this works, some of the FIXME and commented-out lines will need cleanup before merging.` because there is no guarantee that this will work and the original comments may need to be restored. But yes, if the try-job seems to run the test and succeed, this will be removed.

Reply:
ahhh apologies